config/config.go

```diff
@@ -90,6 +92,9 @@ type Config struct {
 
 	// WorkflowRestrictions restricts the controller to executing Workflows that meet certain restrictions
 	WorkflowRestrictions *WorkflowRestrictions `json:"workflowRestrictions,omitempty"`
+
+	//Adding configurable initial delay (for K8S clusters with mutating webhooks) to prevent workflow getting modified by MWC.
+	InitialDelay time.Duration `json:"initialDelay,omitempty"`
```

Comment:
Can we use `metav1.Duration` which parses durations like `10s`

Reply:
@alexec 
Test failed since AddAfter method is expecting only time.Duration

Reply:
```
Config. InitialDelay.Duration
```
???

Reply:
Is this acceptable (to cast to time.Duration)

`wfc.wfQueue.AddAfter(key, time.Duration(wfc.Config.InitialDelay.Seconds()))`

Reply:
`time.Duration` is nano-seconds, so this would be the wrong amount.  I think you can do:

```
wfc.wfQueue.AddAfter(key, wfc.Config.InitialDelay.Duration)
```